macro/Cargo.toml

```diff
@@ -29,6 +29,7 @@ quote = { workspace = true }
 subxt-codegen = { workspace = true, features = ["fetch-metadata"] }
 scale-typegen = { workspace = true }
 polkadot-sdk = { workspace = true, optional = true, features = ["sp-io", "sc-executor-common", "sp-state-machine", "sp-maybe-compressed-blob", "sc-executor"] }
+futures = { workspace = true, features = ["executor"] }
```

Comment:
I guess one reason for `tokio` was just that it was already likely to be included as a dep. I wonder if this adds any new deps to the tree or whether it was also in use somewhere anyways?

Reply:
I don't think so because futures/executor is already enabled by default which compiles for wasm as wel

Reply:
https://github.com/rust-lang/futures-rs/blob/0.3/futures/Cargo.toml#L35